relayer/packetMsgs.go

```diff
@@ -0,0 +1,67 @@
+package relayer
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	chanTypes "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/types"
+)
+
+var ModuleCdc *codec.Codec
+
+func RegisterCodec(cdc *codec.Codec) {
```

Comment:
Shouldn't this live in the Agoric Chain?

Reply:
If there's a way of encoding these packets without using `packetMsgs.go` or depending on third-party stuff, I'm all for it.

Maybe the Route doesn't need to be set?